<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stubby</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>611</width>
      <height>461</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <widget class="QWidget" name="main_tab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>571</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,30,10,20,10">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,0,0,2">
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Network name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="network_name">
               <property name="text">
                <string>Unknown</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Network profile:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="network_profile">
               <property name="text">
                <string>Unknown</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,0,0,0,10">
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="7,3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Service running:   </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="serviceStatus">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="7,3">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Service in use:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="networkStatus">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="7,3">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Connection:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="connectStatus">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTextEdit" name="statusOutput">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>511</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="config_tab">
     <attribute name="title">
      <string>Config</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>581</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="30,10,10">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Network Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Lock config</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Expert mode</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Untrusted (Default)</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>9</y>
             <width>551</width>
             <height>314</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="15,15,95">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="10,10,8">
              <item>
               <widget class="QCheckBox" name="checkBox_3">
                <property name="text">
                 <string>Use as default profile</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Configure Specific Networks</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="10,10,7,0">
              <item>
               <widget class="QCheckBox" name="checkBox_8">
                <property name="text">
                 <string>Encrypt all traffic</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_7">
                <property name="text">
                 <string>Always Autheticate</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_6">
                <property name="text">
                 <string>Validate data</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_4">
                <property name="text">
                 <string>Round robin</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,3,10">
              <item>
               <widget class="Line" name="line_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="5,10,50,5,5">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>DNS Servers</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox">
                  <item>
                   <property name="text">
                    <string>Exclude network provided servers</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Include network provided servers</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Use only network provided servers</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="text">
                   <string>Test the servers</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QPushButton" name="pushButton_11">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_4">
                  <property name="text">
                   <string>Edit</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_6">
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_7">
                  <property name="text">
                   <string>Filter</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableView" name="tableView"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_8">
          <attribute name="title">
           <string>Trusted</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Hostile</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="log_tab">
     <attribute name="title">
      <string>Logs</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Advanced</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>550</y>
      <width>354</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QPushButton" name="revert_do_defaults_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Revert to Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_10">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Discard Edits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="discard_edits_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="runningStatus">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>38</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Running</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="stubbyStatus">
    <property name="geometry">
     <rect>
      <x>446</x>
      <y>38</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QSlider" name="onOffSlider">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>35</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">groove:horizontal {
    border: 1px solid #999999;
    height: 28px;}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Stubby</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>13</y>
      <width>611</width>
      <height>67</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border:none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>tabWidget</zorder>
   <zorder>horizontalLayoutWidget_6</zorder>
   <zorder>runningStatus</zorder>
   <zorder>stubbyStatus</zorder>
   <zorder>label_2</zorder>
   <zorder>onOffSlider</zorder>
   <zorder>label_8</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
